### PR TITLE
update deploy readme

### PR DESCRIPTION
**Pull request by invocamanman**
- update deployment readme

### CODE REVIEW

**Review by krlosMata — APPROVED** [review]
tests are failing

**Review by krlosMata — APPROVED** [review]
:smile_cat:

**Review by stlpruitt314 — DISMISSED** [review]

**Review by eduadiez — APPROVED** [review]
LGTM

File: deployment/deploy_parameters.json.example [hunk 1]
```diff
@@ -14,9 +14,9 @@
     "timelockAddress": "0x617b3a3528F9cDd6630fd3301B9c8911F7Bf063D",
     "minDelayTimelock": 3600,
     "salt": "0x0000000000000000000000000000000000000000000000000000000000000000",
-    "zkEVMDeployerAddress":"",
     "initialZkEVMDeployerOwner" :"0xaddress"
```
Comment (eduadiez) [inline deployment/deploy_parameters.json.example:17]: add `,`

**Review by eduadiez — APPROVED** [review]
LGTM

File: deployment/deploy_parameters.json.example (hunk 1 above)
Comment (stlpruitt314) [reply deployment/deploy_parameters.json.example]: What you mean by add ,

Comment (eduadiez) [reply deployment/deploy_parameters.json.example]: simply add a comma at the end of line to make it a valid json file.

**Pull request merged**